docs/src/main/sphinx/security/ldap.rst

```diff
@@ -243,15 +244,18 @@ Environment configuration
 TLS configuration
 ~~~~~~~~~~~~~~~~~
 
-Access to the Trino coordinator should be through HTTPS when using LDAP
-authentication. The Trino CLI can use either a :ref:`Java Keystore
-<server_java_keystore>` file or :ref:`Java Truststore <cli_java_truststore>`
-for its TLS configuration.
+When using LDAP authentication, access to the Trino coordinator should be
+through :doc:`HTTPS/TLS </security/https>`.
 
-If you are using a keystore file, it can be copied to the client machine and used
-for its TLS configuration. If you are using truststore, you can either use
+The Trino CLI can use a :ref:`PEM certificate <pem_certs>`, a :ref:`Java
```

Comment:
If the cert is a recognized cert from a public CA that is included in Java.. nothing needs to be done. It just works by using https:// in the URL .. same applies for the JDBC driver.. this is a very important convenience factor and should be a huge driver to just buy a proper cert.

Reply:
Maybe we should add a little info around that to the CLI docs

Reply:
Done and done.